example_github_workflow_files/TagBot.yml

```diff
@@ -0,0 +1,16 @@
+name: TagBot
```

Comment:
Could this file get out of date with https://raw.githubusercontent.com/JuliaRegistries/TagBot/master/example.yml? I notice that one has the ssh key and this one doesn't. Why have both?

Reply:
This is kinda confusing, it's not the TagBot file to be used by package repos, it's the file to be used by registries. I should rename it to `TagBot triggers` or something.

Reply:
Yeah a rename might be food